Source/UI/IdentityUIConfiguration.swift

```diff
@@ -89,11 +103,13 @@ public struct IdentityUIConfiguration {
      - parameter tracker: Required implementation of the trackinge events handler
      - parameter localizationBundle: If you have any custom localizations you want to use
      - parameter appName: If you want to customize the app name display in the UI
+     - parameter useBiometrics: If you want to enable biometrics authentication.
     */
     public func replacing(
         theme: IdentityUITheme? = nil,
         isCancelable: Bool? = nil,
         isSkippable: Bool? = nil,
+        useBiometrics: Bool? = UserDefaults.standard.bool(forKey: "Identity.useBiometrics"),
```

Comment:
This can be `nil` like the others and when you set the property it will keep updating of the storage in the setter.